tests/Dockerfile.fedora

```diff
@@ -88,6 +93,7 @@ WORKDIR /
 CMD /bin/bash
 
 FROM base AS full
+ARG PREFIX
```

Comment:
This needs the default value as well.

Reply:
No, having supply the default value multiple times makes no sense. This valueless form is to make that ARG available in this particular context/section/whatever the term.

Reply:
I think it actually didn't work when I didn't set the default value in the second stage (i.e. the variable turned out to be empty) *when* building the second stage (with `--target full`). The Containerfile man page says this:

```
[...]
Note that a second FROM in a Containerfile sets the values associated with an
         Arg variable to nil and they must be reset if they are to be used later in
         the Containerfile
```

Reply:
Having to supply a default several times would be so broken I just refuse to believe they broke it that badly in Containerfile :smile: